contrib/macdeploy/README.md

```diff
@@ -56,29 +56,14 @@ The `sha256sum` should be `c0c2e7bb92c1fee0c4e9f3a485e4530786732d6c6dd9e9f418c28
 
 ## Deterministic macOS App Notes
 
-macOS Applications are created in Linux by combining a recent `clang` and the Apple
-`binutils` (`ld`, `ar`, etc).
+macOS Applications are created in Linux by combining a recent LLVM (`clang`,
+`lld` + binutils)
```

Comment:
nit: As the word 'combining' has only one object, rather than two as it was before, it seems reasonable to drop it altogether.

Reply:
Given this doesn't yet work, and we might be reithinking the approach (which might render this text irrelevant, I'll just leave stuff like this for now.